packages/features/ee/teams/components/TeamList.tsx

```diff
@@ -44,8 +48,69 @@ export default function TeamList(props: Props) {
     deleteTeamMutation.mutate({ teamId });
   }
 
+  if (!user) return null;
+  // TODO: Derive it
+  const hasOrgPlan = false;
```

Comment:
Do we still need to derive this? Or is this a future update todo

Reply:
I have removed it for now. Yet to push the change.